src/modules/material-requests/material-requests.types.ts

```diff
@@ -30,6 +31,7 @@ export interface MaterialRequest {
 	requesterUserGroupLabel?: string;
 	requesterUserGroupDescription?: string;
 	maintainerLogo?: string;
+	contactMail?: any;
```

Comment:
From the fetch code i would assume this type is string | null:
```typescript
			contactMail:
				(grapqhQLMaterialRequest as FindMaterialRequestsQuery['app_material_requests'][0])
					?.object?.maintainer?.information?.contact_point || null,
```

```suggestion
	contactMail?: string | null;
```